packages/@ember/-internals/routing/lib/ext/controller.ts

```diff
@@ -169,7 +169,7 @@ ControllerMixin.reopen({
     deprecateTransitionMethods('controller', 'transitionToRoute');
 
     // target may be either another controller or a router
-    let target = get(this, 'target');
+    let target = get(this, 'target') as any;
```

Comment:
I was going to ask about typing this and then saw that it's a mixin and :sob:

Can we possibly do a double cast, `as any as (Route | Controller)`?

Reply:
I'll switch to an assert.